utils/measure/.env.dist

```diff
@@ -8,32 +8,36 @@ LIGHT_CONTROLLER=hue
 LOG_LEVEL=INFO
 
 # time between changing the light params and taking the measurement
-SLEEP_TIME=2
+SLEEP_TIME=3
 
 # time between taking multiple samples for the same light setting
-SLEEP_TIME_SAMPLE=1
+SLEEP_TIME_SAMPLE=3
 
 # additional time to wait for each significant change in hue
-SLEEP_TIME_HUE=5 
+SLEEP_TIME_HUE=2 
 
 # additional time to wait for each significant change in saturation
-SLEEP_TIME_SAT=10
+SLEEP_TIME_SAT=2
 
 # additional time to wait for each significant change in color temperature
-SLEEP_TIME_CT=10
+SLEEP_TIME_CT=1
 
 # Set to a value higher than 1 to take multiple samples to reduce noise
-SAMPLE_COUNT=1
+SAMPLE_COUNT=2
 
 # Step sizes for HS mode loop. You can increase these to improve the duration of measure session
 # Don't set higher than duplicate these values!
-HS_BRI_STEPS=10
-HS_HUE_STEPS=2000
-HS_SAT_STEPS=10
+
+# 5 values
+HS_BRI_STEPS=51
```

Comment:
I would suggest to set this to 32 (8 steps), which is a good compromise between the old amount of steps (13) and the amount proposed in this PR.
I approve the changes in Hue and Sat steps. 

Also the python config needs to be changed, which constrains these values to a max.
```
HS_BRI_STEPS = min(config("HS_BRI_STEPS", default=10, cast=int), 20)
HS_HUE_STEPS = min(config("HS_HUE_STEPS", default=2000, cast=int), 4000)
HS_SAT_STEPS = min(config("HS_SAT_STEPS", default=10, cast=int), 20)
``` 

But when we don't allow the values to be higer than 32, 2731 and 32 I would suggest to remove the env variables all together and just set these as fixed values in `measure.py`

Reply:
How about changing them to something like "precision"? 

So 1 is the default precision and the user can choose to set it as low as 0.5 or as high as 4?

This would take out the confusion with these steps. :)